.circleci/config.yml

```diff
@@ -216,6 +216,9 @@ jobs:
     - store_artifacts:
         path: plume.tar.gz
         destination: plume.tar.gz
+    - store_artifacts:
+        path: wasm.tar.gz
+        destination: wasm.tar.gz
```

Comment:
and as such, I'd just call it static.tar.gz

(should this have a version number)